### ISSUE TITLE
Graphic Annoations

### ISSUE THREAD

**Issue by johanna-einsiedler** [idyll-lang/idyll#728]
**Is your feature request related to a problem? Please describe.**
Not a real problem, but would be nice if there was a possibility to include graphics in annotations.

**Describe the solution you'd like**
A way to include a picture in the annotation component. I.e. if the user hovers over a word that a small picture pops up instead of a text like in the standard annotation component.

**Comment by mathisonian** [comment]
@jasenlo123 would you be interested in tackling this?

**Comment by jasenlo123** [comment]
Thanks for bringing this to my attention @mathisonian. 

I'll have time to work on this in the next two weeks @johanna-einsiedler. Cheers!

**Comment by jasenlo123** [comment]
hi @mathisonian, here's the PR that allows for including graphics in annotations!
https://github.com/idyll-lang/idyll/pull/730

**Comment by mathisonian** [comment]
This is addressed in #730 and available in the latest release. See the updated docs here: https://idyll-lang.org/docs/component/annotation.

The basic usage is updated to:

```
[Annotation] 
  [Graphic] Anything in here will be shown in the popup. Including, e.g. an image: [Image src:"..." /] [/Graphic]

  Text outside of the graphic tag will be displayed inline. 
[/Annotation]
```
